syft/core/frameworks/torch/tensor.py

```diff
@@ -1028,7 +1028,8 @@ def handle_call(cls, syft_command, owner):
         self_ = tensor_command["self"] if has_self else None
 
         if attr == "end_get":
-            response = self_.get()
+            # For end_get use the tensor in the original Syft command
+            response = syft_command["self"].get()
```

Comment:
I realize this may not be the best solution but given my limited knowledge of the codebase this is what I came up with. Will be happy to modify, but at least I think we have an issue that needs resolving here, one way or another.

Reply:
I literally tried to solve this yesterday and if your solution works it's VASTLY superior! Well done! (will check in a bit)